--- numpy-1.16.3/numpy/distutils/intelccompiler.py	2019-02-21 17:33:40.000000000 -0800
+++ patch-files/intelccompiler.py	2019-05-25 16:18:58.575362543 -0700
@@ -60,7 +60,7 @@
 
         v = self.get_version()
         mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 '
+        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 -xHost'
                        '-fomit-frame-pointer -{}').format(mpopt)
         compiler = self.cc_exe
 
