--- numpy-1.15.2/numpy/distutils/intelccompiler.py	2018-09-17 20:25:12.000000000 -0700
+++ patch-files/intelccompiler.py	2018-09-28 18:15:19.146784620 -0700
@@ -60,7 +60,7 @@
 
         v = self.get_version()
         mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 '
+        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 -xhost'
                        '-fomit-frame-pointer -{}').format(mpopt)
         compiler = self.cc_exe
 
