--- numpy-1.9.1/numpy/distutils/intelccompiler.py	2014-10-26 07:36:14.000000000 -0700
+++ patch-files/intelccompiler.py	2014-11-20 07:04:08.899936025 -0800
@@ -11,7 +11,7 @@
 
     def __init__ (self, verbose=0, dry_run=0, force=0):
         UnixCCompiler.__init__ (self, verbose, dry_run, force)
-        self.cc_exe = 'icc -fPIC'
+        self.cc_exe = 'icc -O3 -g -fPIC -fp-model strict -fomit-frame-pointer -openmp -xhost'
         compiler = self.cc_exe
         self.set_executables(compiler=compiler,
                              compiler_so=compiler,
