--- numpy-1.17.0/numpy/distutils/intelccompiler.py	2019-06-15 17:03:54.000000000 -0700
+++ patch-files/intelccompiler.py	2019-07-30 06:25:22.341386316 -0700
@@ -60,7 +60,7 @@
 
         v = self.get_version()
         mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 '
+        self.cc_exe = ('icc -m64 -fPIC -fp-model strict -O3 -xHost'
                        '-fomit-frame-pointer -{}').format(mpopt)
         compiler = self.cc_exe
 
