--- numpy-1.9.1/numpy/distutils/fcompiler/intel.py	2014-10-26 07:36:14.000000000 -0700
+++ patch-files/intel.py	2014-11-20 07:08:10.291935089 -0800
@@ -117,7 +117,7 @@
         return ['-xhost -openmp -fp-model strict']
 
     def get_flags_arch(self):
-        return []
+        return ['-fPIC', '-openmp']
 
 # Is there no difference in the version string between the above compilers
 # and the Visual compilers?
