--- numpy-1.15.2/numpy/distutils/fcompiler/intel.py	2018-09-17 20:25:12.000000000 -0700
+++ patch-files/intel.py	2018-09-28 18:18:44.895798214 -0700
@@ -124,10 +124,10 @@
     def get_flags_opt(self):  # Scipy test failures with -O2
         v = self.get_version()
         mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        return ['-fp-model strict -O1 -{}'.format(mpopt)]
+        return ['-fPIC -fp-model strict -O1 -{}'.format(mpopt)]
 
     def get_flags_arch(self):
-        return ['']
+        return ['-fPIC']
 
 # Is there no difference in the version string between the above compilers
 # and the Visual compilers?
