--- numpy-1.16.3/numpy/distutils/fcompiler/intel.py	2019-02-21 17:33:40.000000000 -0800
+++ patch-files/intel.py	2019-05-25 16:22:08.636366464 -0700
@@ -127,10 +127,10 @@
     def get_flags_opt(self):  # Scipy test failures with -O2
         v = self.get_version()
         mpopt = 'openmp' if v and v < '15' else 'qopenmp'
-        return ['-fp-model strict -O1 -{}'.format(mpopt)]
+        return ['-fPIC -fp-model strict -O1 -{}'.format(mpopt)]
 
     def get_flags_arch(self):
-        return ['']
+        return ['-fPIC']
 
 # Is there no difference in the version string between the above compilers
 # and the Visual compilers?
