--- site.cfg.example	2017-03-23 20:17:13.000000000 -0700
+++ patch-files/site.cfg.example	2017-03-23 20:19:20.000000000 -0700
@@ -82,8 +82,8 @@
 # This is a good place to add general library and include directories like
 # /usr/local/{lib,include}
 #
-#[ALL]
-#library_dirs = /usr/local/lib
+[ALL]
+library_dirs = /usr/lib64
 #include_dirs = /usr/local/include
 #
 
@@ -161,6 +161,12 @@
 # [mkl]
 # library_dirs = /opt/intel/mkl/9.1.023/lib/32/
 # lapack_libs = mkl_lapack
+ROLL_COMMENTMKL [mkl]
+ROLL_COMMENTMKL library_dirs = ROLL_MKLLIB
+ROLL_COMMENTMKL mkl_libs = mkl_rt
+ROLL_COMMENTMKL include_dirs =  ROLL_MKLROOT/include
+ROLL_COMMENTMKL lapack_libs =  mkl_rt
+
 #
 # For 10.*, on 32 bits machines:
 # [mkl]
