--- numpy-1.17.0/site.cfg.example	2019-07-26 08:10:50.000000000 -0700
+++ patch-files/site.cfg.example	2019-07-30 07:13:28.458625019 -0700
@@ -83,11 +83,17 @@
 # This is a good place to add general library and include directories like
 # /usr/local/{lib,include}
 #
-#[ALL]
-#library_dirs = /usr/local/lib
-#include_dirs = /usr/local/include
+[ALL]
+library_dirs = /usr/lib64
+include_dirs = /opt/python/include
 #
 
+ROLL_COMMENTMKL [mkl]
+ROLL_COMMENTMKL library_dirs = ROLL_MKLLIB
+ROLL_COMMENTMKL mkl_libs = mkl_rt
+ROLL_COMMENTMKL include_dirs =  ROLL_MKLROOT/include
+ROLL_COMMENTMKL lapack_libs =  mkl_rt
+
 # ATLAS
 # -----
 # ATLAS is an open source optimized implementation of the BLAS and LAPACK
