--- numpy-1.16.3/numpy/distutils/system_info.py	2019-04-21 12:06:28.000000000 -0700
+++ patch-files/system_info.py	2019-05-25 16:28:05.216373582 -0700
@@ -290,13 +290,13 @@
     default_x11_include_dirs = ['/usr/X11R6/include', '/usr/X11/include',
                                 '/usr/include']
 
-    if os.path.exists('/usr/lib/X11'):
-        globbed_x11_dir = glob('/usr/lib/*/libX11.so')
-        if globbed_x11_dir:
-            x11_so_dir = os.path.split(globbed_x11_dir[0])[0]
-            default_x11_lib_dirs.extend([x11_so_dir, '/usr/lib/X11'])
-            default_x11_include_dirs.extend(['/usr/lib/X11/include',
-                                             '/usr/include/X11'])
+#   if os.path.exists('/usr/lib/X11'):
+#       globbed_x11_dir = glob('/usr/lib/*/libX11.so')
+#       if globbed_x11_dir:
+#           x11_so_dir = os.path.split(globbed_x11_dir[0])[0]
+#           default_x11_lib_dirs.extend([x11_so_dir, '/usr/lib/X11'])
+#           default_x11_include_dirs.extend(['/usr/lib/X11/include',
+#                                            '/usr/include/X11'])
 
     import subprocess as sp
     tmp = None
